.github/actions/setup-microceph/action.yml

```diff
@@ -51,17 +51,33 @@ runs:
           }
           trap cleanup ERR HUP INT TERM
 
+          ephemeral_disk="${{ steps.free_ephemeral_disk.outputs.ephemeral_disk }}"
+          sudo parted "${ephemeral_disk}" --script mklabel gpt
+          sudo parted "${ephemeral_disk}" --script mkpart primary 0% 33%
+          sudo parted "${ephemeral_disk}" --script mkpart primary 33% 66%
+          sudo parted "${ephemeral_disk}" --script mkpart primary 66% 100%
+
+          disk1="$(losetup -f)"
+          sudo losetup "${disk1}" "${ephemeral_disk}1"
+          disk2="$(losetup -f)"
+          sudo losetup "${disk2}" "${ephemeral_disk}2"
+          disk3="$(losetup -f)"
+          sudo losetup "${disk3}" "${ephemeral_disk}3"
```

Comment:
I take it that MicroCeph still cannot take partitions directly right? How about adding a link to https://github.com/canonical/microceph/issues/251 in a comment?